contracts/core/extensions/CoreIssuance.sol

```diff
@@ -139,17 +165,33 @@ contract CoreIssuance is
         uint _toWithdraw
     )
         external
-        isValidSet(_set)
-        isPositiveQuantity(_quantity)
```

Comment:
Let's double check whether these checks are actually necessary, what do cases where these don't hold true expose us to?

Reply:
One reason I ask is the data type is already uint.